ducktape/services/service.py

```diff
@@ -150,6 +150,12 @@ def wait(self):
         """
         pass
 
+    def wait_node(self, node, timeout_sec=None):
```

Comment:
@hachikuji One thing to keep in mind is that adding the method and docs kind of implies that every class needs to implement it. Maybe that's not a problem in this case since you pretty much always need this functionality? But its worth keeping in mind that putting it in the `Service` class basically commits all `Service` implementations to honor the interface. (Although, at a minimum, in this case it seems that if you wanted to let people opt out this is at least a better default impl than raising a `NotImplementedError`.)